portugues/colabora/semdiferencas/pentefino/pentefino.md

```diff
@@ -2,4 +2,4 @@ Depois de passar horas analisando e depurando, você
 
 [Descobre um erro de lógica](../../comdiferencas/diff/diff.md)
 
-[Já não consegue raciocinar e fica babando no teclado de tanto sono](../../colaborar.md)
```

Comment:
Okay, it is hard for me to tell how boring it may be, since I don't speak Portuguese. I'll go ahead and merge this, then.